txpool/slot_gauge.go

```diff
@@ -27,6 +31,12 @@ func (g *slotGauge) decrease(slots uint64) {
 	atomic.AddUint64(&g.height, ^(slots - 1))
 }
 
+// highPressure checks if the gauge level
+// is higher than the highPressureMark (0.8 * max)
```

Comment:
Why do you write actual value (0.8) instead of highPressureMark?

Reply:
[Fixed](https://github.com/0xPolygon/polygon-edge/pull/689/commits/8529040d699d5df794073eb115f9f41d58514291)